docs(readme): add quest list manager basic information (#31)
Invasor-de-Fronteiras/mhfrontier-cq-tool: README.md

Nice! Now do the quest changes as your desire and save it.

> ### Manager questlist

Reviewer: ```suggestion
### Quest List Manager
```

To show quest in game it's necessary to add quest in questlist, for its you need to change the selected tool **QuestEditor** to **QuestlistEditor**.

![SelectTool](./screenshots/select_tool.png)

Now you click in **Load Folder** and select questlist folder, after this you can manager your quests in menu **Questlist** and add quests clicking on **Add Quest from file**.

After all you can save questlist in **Save Questlist**.

> ### Manager questlist in 9.1

Reviewer: ```suggestion
### Manager Quest List in [Erupe 9.1](https://github.com/ZeruLight/Erupe/releases/tag/v9.1.0)
```

> ### Manager Quest List in [Erupe 9.1](https://github.com/ZeruLight/Erupe/releases/tag/v9.1.0)

Reviewer: ```suggestion
#### Quest List Manager for [Erupe 9.1](https://github.com/ZeruLight/Erupe/releases/tag/v9.1.0)
```

> After update 9.1 from [Erupe CE](https://github.com/ZeruLight/Erupe) the questlist files are removed and now we need to export a template from questfile to folder **/bin/events**.

Reviewer: ```suggestion
After update [9.1](https://github.com/ZeruLight/Erupe/releases/tag/v9.1.0), the quest list files were removed and now we require to export a template from the quest file to folder **/bin/events**.
```
